addons/point_of_sale/models/report_sale_details.py

```diff
@@ -350,9 +350,10 @@ def _get_product_total_amount(self, line):
     def _get_products_and_taxes_dict(self, line, products, taxes, currency):
         key2 = (line.product_id, line.price_unit, line.discount)
         key1 = line.product_id.product_tmpl_id.pos_categ_ids[0].name if len(line.product_id.product_tmpl_id.pos_categ_ids) else _('Not Categorized')
+        precision = self.env['decimal.precision'].precision_get('Product Unit of Measure')
```

Comment:
```suggestion
        precision = self.env['decimal.precision'].precision_get('Product Unit')
```

I believe `Product Unit of Measure` is no longer there starting saas-18.2.